intents/oversights/simpsons_paradox.py

```diff
@@ -0,0 +1,292 @@
+
+def simpsons_paradox(table, metric, all_dimensions, slice_compare_column, 
+                            slice1, slice2, summary_operator, **kwargs):
+    """This function will implement the simpson's-paradox debaising
+
+    Args:
+        table: Type-pandas.dataframe
+            It has the contents of the csv file
+        metric: Type-string
+            It is the name of the column according to which we will do grouping,
+            summary operator is applied on metric. Metric could a column
+            containing strings, if we are applying count operator on it.
+        dimensions: Type-list of str or None
+            It is the name of column we want.
+            In query:'compare batsman A and B according to total_runs',
+             dimension is 'batsman'. we group by dimensions.
+        all_dimension: Type-list of str
+        	It contains list of all dimensions
+        slice_compare_column: Type-string
+            name of the slice-compare column.
+        slice1: Type-string
+            the first value of comparision
+        slice2: Type-string
+            the second value of comparision
+        date_range: Type-tuple
+            Tuple of start_date and end_date
+        date_column_name: Type-str
+            It is the name of column which contains date
+        date_format: Type-str
+            It is required by datetime.strp_time to parse the date in the format
+            Format Codes
+https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior
+        slices: Type-List of tuples
+            Tuple represents the conditon to keep the row.
+            (column_name, filter, value)
+            column_name - is the value of the column that the
+            condition is applied upon.
+            filter - Filters enum members, ex. Filters.IN
+        summary_operator: Type-summary_operators enum members
+            It denotes the summary operator, after grouping by dimensions.
+            ex. SummaryOperators.MAX, SummaryOperators.SUM
+
+    Note-summary_operator is always applied on metric column passed,
+         and only when grouping is done
+
+    Returns:
+        return a list of dictionary where each dictionary represent
+               a debiasing suggestion according to the new column.
+    """
+
+    date_column_name = kwargs.get('date_column_name', 'date')
+    date_range = kwargs.get('date_range', None)
+    date_format = kwargs.get('date_format', '%Y-%m-%d')
+
+    slices = kwargs.get('slices', None)
+
+    dimensions = kwargs.get('dimensions', None)
+
+    table = aspects.apply_date_range(table, date_range,
+                                     date_column_name, date_format)
+
+    slice_list = []
+    if slices is not None:
+        slice_list = slices.copy()
+    slice_list.append((slice_compare_column, Filters.IN, [slice1, slice2]))
+
+    table = aspects.slice_table(table, slice_list)
+
+    """removing all metric column except the one by which we will
```

Comment:
For comments such as these, don't use docstrings, use # instead - see here: https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments

Reply:
Fixed

Reply:
@11chankun  maybe you could also use # instead of docstrings in the previously submitted files as well. (I did not notice this while reviewing the time compare intent) 
https://github.com/11chankun/debaised-analysis/blob/master/intents/time_compare.py#L155-L156